frontstage/templates/secure-messages/conversation-view.html

```diff
@@ -15,7 +15,7 @@
         {% call onsPanel({
             "classes": "ons-u-mb-s"
         }) %}
-            <p class="ons-u-fs-r ons-u-mb-xs">This conversation has now been closed. <br>
+<p class="ons-u-fs-r ons-u-mb-xs">This conversation has now been closed. </p>
```

Comment:
Umm, this one might be wrong still?  Adding the closing p is good, but you removed the <br>.
Also this needs to be indented back to how it was as it looks weird with it being all the way to the left